src/templates/kit/Tokens.js

```diff
@@ -0,0 +1,231 @@
+import React, { useCallback, useState } from 'react'
+import {
+  Button,
+  EthIdenticon,
+  Field,
+  GU,
+  Help,
+  IconPlus,
+  IconTrash,
+  Info,
+  RADIUS,
+  TextInput,
+  isAddress,
+  useTheme,
+} from '@aragon/ui'
+import { Header, PrevNextFooter } from '.'
+
+function Tokens({ back, data, fields, next, screenIndex, screens }) {
+  const theme = useTheme()
+
+  const [tokenName, setTokenName] = useState(data.tokenName || '')
+  const [tokenSymbol, setTokenSymbol] = useState(data.tokenSymbol || '')
+  const [members, setMembers] = useState(data.members || [''])
+
+  const handleTokenNameChange = useCallback(event => {
+    setTokenName(event.target.value)
+  }, [])
+
+  const handleTokenSymbolChange = useCallback(event => {
+    setTokenSymbol(event.target.value)
+  }, [])
+
+  const addMember = useCallback(() => {
+    setMembers(members => [...members, ''])
```

Comment:
Should we double check these are unique?
![image](https://user-images.githubusercontent.com/3072458/64345889-6b8d4880-cff1-11e9-96b9-f6400148258f.png)

Reply:
I don't think anyone should type an ethereum address but this input's lose focus after the first char has been inputted.

Reply:
> I don't think anyone should type an ethereum address but this inputs lose focus after the first char has been inputted.

Yes I am going to fix the focus management all around the onboarding, this is not done yet.

Reply:
> Should we double check these are unique?

Yes, I need to add validation in general (invalid addresses, duplicate address, etc.).